### ISSUE TITLE
exec not working with cloned perls

### ISSUE THREAD

**Issue by stevieb9** [stevieb9/berrybrew#183]
See https://github.com/dnmfarrell/berrybrew/issues/23

**Comment by stevieb9** [comment]
After changing `custom_exec` from `false` (the default) to `true` in the `data\config.json` file, it works as expected (work on dnmfarrell/berrybrew#23):

```
c:\repos\berrybrew>berrybrew clone 5.10.1_32 clone-test
Successfully registered clone-test
Successfully installed custom perl 'clone-test'

c:\repos\berrybrew>berrybrew exec perl -E "say 'hi'"

perl-5.10.1_32
==============
hi
perl-clone-test
==============
hi
```

**Comment by stevieb9** [comment]
Closes dnmfarrell/berrybrew#23
